contracts/src/instances/ezeth/EzETHTarget0.sol

```diff
@@ -19,6 +19,12 @@ import { EzETHBase } from "./EzETHBase.sol";
 contract EzETHTarget0 is HyperdriveTarget0, EzETHBase {
     using SafeERC20 for ERC20;
 
+    /// @dev The instance's name.
+    string internal constant NAME = "EzETHHyperdrive";
+
+    /// @dev The instance's version.
+    string internal constant VERSION = "v1.0.0";
```

Comment:
these version strings are going to be tedious update depending on what we consider a version rev. does this rev when the internal hyperdrive code revs?